lib/std/os/linux/test.zig

```diff
@@ -108,3 +108,22 @@ test "user and group ids" {
     expectEqual(linux.getauxval(elf.AT_EUID), linux.geteuid());
     expectEqual(linux.getauxval(elf.AT_EGID), linux.getegid());
 }
+
+test "posix_fadvise" {
+    const tmp_file_name = "temp_posix_fadvise.txt";
+    var file = try fs.cwd().createFile(tmp_file_name, .{});
+    defer {
+        file.close();
+        fs.cwd().deleteFile(tmp_file_name) catch {};
+    }
+
+    const stat = try file.stat();
+
+    const ret = linux.posix_fadvise(
+        file.handle,
+        0,
+        @bitCast(i64, stat.size),
```

Comment:
This part is a bit iffy as passing 0 here means "up to the end" and this file is empty...
I'd fill the file with garbage and then pass 0 here.

Reply:
Addressed in the last commit.

Reply:
There's no need to write random data, you can inflate the file by writing garbage or using fallocate.

Reply:
Ok, I've changed it to just write an undefined buffer.